### PR TITLE
liveimage-mount: Add support for OverlayFS overlays.

### PR DESCRIPTION
**Pull request by FGrose**
Also, simplify util-linux wrapper functions.
And further Python3 compatibility.

### CODE REVIEW

File: tools/liveimage-mount [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/python
-# coding: utf-8
+# coding: utf_8
```
Comment (Conan-Kudo) [inline tools/liveimage-mount]: This is not a valid reference, per [PEP 263](https://www.python.org/dev/peps/pep-0263/).

File: tools/liveimage-mount [hunk 2]
```diff
@@ -141,56 +142,55 @@ def call(*popenargs, **kwargs):
...
-    out, err, p, environ = '', '', None, None
+    out, err, p, environ = b'', b'', None, None
     if env is not None:
         environ = os.environ.copy()
         environ.update(env)
     try:
         p = subprocess.Popen(args, stdin=subprocess.PIPE, cwd=cwd, env=environ,
                              stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-        out, err = p.communicate(stdin)
+        out, err = p.communicate(stdin.encode('utf_8'))
```
Comment (Conan-Kudo) [inline tools/liveimage-mount]: Needs to be `utf-8`.

File: tools/liveimage-mount [hunk 3]
```diff
@@ -141,56 +142,55 @@ def call(*popenargs, **kwargs):
...
         if p.returncode != 0 and raise_err:
-            raise LiveImageMountError(u"Error in call:\n'%s'\nenviron: '%s'\n"
-                               "stdout: '%s'\nstderr: '%s'\nreturncode: '%s'" %
-                                      (args, environ, out, err, p.returncode))
+            raise LiveImageMountError('Error in call:\n%s\nenviron: %s\n'
+                                    'stdout: %s\nstderr: %s\nreturncode: %s' %
+                                       (args, environ, out, err, p.returncode))
     finally:
-        return out, err
+        return out.decode('utf_8'), err.decode('utf_8'), p.returncode
```
Comment (Conan-Kudo) [inline tools/liveimage-mount]: Needs to be `utf-8`.

File: tools/liveimage-mount [hunk 4]
```diff
@@ -700,19 +786,22 @@ def main():
...
 
+        a = X
+        if overlayfs:
+            a = Y
         environ = os.environ.copy()
-        if X:
+        if a:
             # This is overridden if PS1 is set in /root/.bashrc.
-            environ['PS1'] = X + '-[\u@\h \W]\$ '
+            environ['PS1'] = a.encode('utf_8') + b'-[\u@\h \W]\$ '
```
Comment (Conan-Kudo) [inline tools/liveimage-mount]: Needs to be `utf-8`.

Comment (FGrose) [inline tools/liveimage-mount]: I used this reference,
https://docs.python.org/3.6/library/codecs.html?highlight=utf_8#standard-encodings
where utf-8 is probably the most popular alias for utf_8.

Comment (Conan-Kudo) [inline tools/liveimage-mount]: The reason for that is that editors read `utf-8` but don't necessarily recognize `utf_8`.

**Comment by Conan-Kudo** [comment]
@FGrose Could you take a look at addressing the feedback?

**Comment by FGrose** [comment]
This commit should address issue #73 and your feedback.

**Comment by Conan-Kudo** [comment]
Merged, thanks!

**Pull request merged**